src/components/Typeahead/Typeahead.tsx

```diff
@@ -44,20 +61,32 @@ const Typeahead = (props: Props) => {
   }
 
   return (
-    <AsyncTypeahead
-      id={id}
-      labelKey={searchAccessor}
-      options={options}
-      placeholder={placeholder}
-      isLoading={isLoading}
-      minLength={minLength}
-      onSearch={search}
-      onChange={onChange}
-      renderMenuItemChildren={renderMenuItemChildren}
-      defaultSelected={selectedValues}
-      disabled={disabled}
-      isInvalid={isInvalid}
-    />
+    <>
+      <AsyncTypeahead
+        id={id}
+        labelKey={searchAccessor}
+        options={options}
+        placeholder={placeholder}
+        isLoading={isLoading}
+        minLength={minLength}
+        onSearch={search}
+        onChange={onChange}
+        renderMenuItemChildren={renderMenuItemChildren}
+        defaultSelected={selectedValues}
+        disabled={disabled}
+        isValid={isValid}
+        isInvalid={isInvalid}
+      />
+      {feedback !== undefined && (
+        <div
+          className={`text-left ml-2 mt-1 text-small ${
+            isValid ? 'text-success' : isInvalid ? 'text-danger' : undefined
```

Comment:
I noticed that in storybook, the text is not green when it is valid.

Reply:
It's the same as the valid feedback text for `DateTimePicker` and `TextInput`, it's the success color. I see `TextInput` doesn't have the green border when it's valid, either. Its border changes to the success color when it's valid.

It looks like custom CSS was added to `DateTimePicker` because it doesn't automatically support the valid/invalid prop styling, but probably the green border should be changed to the success color both in `DateTimePicker` and `Typeahead`.

Reply:
Yeah, we should probably change the CSS to match globally

Reply:
I should be able to use SASS variables.

Reply:
I ended up removing both custom CSS files and just applied `.is-valid` and `.is-invalid` classes to `DateTimePicker`. Now text and borders should style using the theme colors.